app/api/items/fetch_active_attempt.go

```diff
@@ -13,6 +13,60 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/token"
 )
 
+// swagger:operation PUT /items/{item_id}/active-attempt items itemActiveAttemptFetch
+// ---
+// summary: Fetch an active attempt
+// description: >
+//
+//   Returns a task token for the active attempt for the given item. If the active attempt doesn't exist,
+//   the service creates it. A row in `users_items` is also created if needed.
+//
+//
+//   Depending on the `items.bHasAttempts` the active attempt is linked to the user's self group (if `items.bHasAttempts`=0)
+//   or to the user’s team (if `items.bHasAttempts`=1). The user’s team is a user's parent group with `groups.idTeamItem`
+//   pointing to one of the item's ancestors or the item itself.
```

Comment:
It returns "forbidden" when should be in team but the user has no team. Should be written in specs.

Reply:
fixed